web_server/users/static/users/css/style.css

```diff
@@ -0,0 +1,73 @@
+.background-img {
+	background: url('/static/users/images/background.jpg');
+	background-size:100% 100%;
+}
+
+.item-center {
+	margin-top: 10px;
+	position: fixed;
+	left: 50%;
+	top: 50%;
+	-webkit-transform: translateX(-50%) translateY(-50%);
+}
+
+.login-l {
+	float: left;
+	width: 70%;
+}
+
+.login-r {
+	margin-left: 72%;
+	width: 20%;
+	position: fixed;
+	top: 50%;
+	-webkit-transform: translateY(-50%);
+}
+
+.register {
+	margin-top: 20px;
+	width: 35%;
+	margin-bottom: 40px;
+}
+
+.register-title {
+	margin-top: 10px;
+	margin-bottom: 40px;
+	text-align: center;
+}
+
+.profile-part-I {
```

Comment:
These class names are hard to guess what they are for. I would look at the page but due to a database issue I cannot get this branch to run. These classes should get named to what they mean (e.g. profile-icon, or profile-content) or they could be separated out so that the outer .profile block has an inner .icon block and the rule could be changed to ".profile > .icon"